src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -62,15 +62,15 @@ protected function __construct()
     {
     }
 
-    public static function export() : void
+    public static function export()
```

Comment:
This is indeed supposed to be `void`

Reply:
This method inherits from a method that has a non-`void` return type

Reply:
http://php.net/manual/en/reflector.export.php

Reply:
If the parent were user-defined, a `void` return type would be a fatal error: https://3v4l.org/BXtvR